pilot/cmd/pilot-agent/status/server_test.go

```diff
@@ -737,6 +729,8 @@ func TestAppProbe(t *testing.T) {
 		defer cancel()
 		go server.Run(ctx)
 
+		// make sure the upstreamLocalAddress would not be nil
+		server.upstreamLocalAddress = &net.TCPAddr{IP: net.ParseIP("127.0.0.1")}
```

Comment:
how come this is needed now?

Reply:
I used `upstreamLocalAddress` in my first commit, and forgot to remove it. I will remove it.